### ISSUE TITLE
Svelte TypeError: Failed to fetch dynamically imported module

### ISSUE THREAD

**Issue by givanse** [givanse/givanse#6]
Loading the app after `npm run dev` fails with:

```
Uncaught (in promise) TypeError: Failed to fetch dynamically imported module: http://localhost:3000/src/routes/__layout.svelte
```

But `npm run build && npm run preview` works fine.

**Comment by givanse** [comment]
This commit https://github.com/givanse/givanse/commit/d2b6e2528954d5f800921b95bd3c5105d58a8269 is the culprit, but it fixes issue #5.
> svelte dynamic route won't load on page refresh

**Comment by givanse** [comment]
Reverted the dependencies to what they were before this error started happening, PR https://github.com/givanse/givanse/pull/7
